IBAnimatable/MaskDesignable.swift

```diff
@@ -37,6 +39,11 @@ public extension MaskDesignable where Self: UIView {
     layer.cornerRadius = ceil(min(bounds.width, bounds.height))/2
   }
   
+  public func maskPolygon() {
+    let polygonPath = maskPolygonBezierPath(Int(arc4random() % 10 + 3))
```

Comment:
I think `arc4random_uniform` is a little bit better in this case. It would be `let polygonPath = maskPolygonBezierPath(Int(arc4random_uniform(10) + 3))`, please have a look at http://nshipster.com/random/ 

Will create a doc/an issue for "customise the parameters" and let us vote.